data/aloha-rubyconf/aloha-rubyconf-2012/videos.yml

```diff
@@ -145,6 +145,7 @@
     - Charles Nutter
   event_name: Aloha RubyConf 2012
   published_at: "2012-10-08"
+  slides_url: https://fr.slideshare.net/CharlesNutter/aloha-rubyconf-2012-jruby
```

Comment:
```suggestion
  slides_url: https://speakerdeck.com/headius/aloha-rubyconf-2012-jruby
```

Reply:
Nothing wrong with the one you put 🙈 I just swapped it for Speakerdeck since we can embed those.

Reply:
Good to know!